Only in /root/linux-5.19.9/: ..vmlinux.export.o.cmd
Only in /root/linux-5.19.9/: .Module.symvers.cmd
Only in /root/linux-5.19.9/: .config
Only in /root/linux-5.19.9/: .config.old
Only in /root/linux-5.19.9/: .missing-syscalls.d
Only in /root/linux-5.19.9/: .modules-only.symvers.cmd
Only in /root/linux-5.19.9/: .modules.order.cmd
Only in /root/linux-5.19.9/: .tmp_System.map
Only in /root/linux-5.19.9/: .tmp_vmlinux.kallsyms1
Only in /root/linux-5.19.9/: .tmp_vmlinux.kallsyms1.S
Only in /root/linux-5.19.9/: .tmp_vmlinux.kallsyms1.o
Only in /root/linux-5.19.9/: .tmp_vmlinux.kallsyms2
Only in /root/linux-5.19.9/: .tmp_vmlinux.kallsyms2.S
Only in /root/linux-5.19.9/: .tmp_vmlinux.kallsyms2.o
Only in /root/linux-5.19.9/: .version
Only in /root/linux-5.19.9/: .vmlinux.cmd
Only in /root/linux-5.19.9/: .vmlinux.export.c
Only in /root/linux-5.19.9/: .vmlinux.export.o
Only in /root/linux-5.19.9/: .vmlinux.o.cmd
Only in /root/linux-5.19.9/: .vmlinux.objs
Common subdirectories: /root/patch/linux-5.19.9/Documentation and /root/linux-5.19.9/Documentation
Common subdirectories: /root/patch/linux-5.19.9/LICENSES and /root/linux-5.19.9/LICENSES
diff -u /root/patch/linux-5.19.9/Makefile /root/linux-5.19.9/Makefile
--- /root/patch/linux-5.19.9/Makefile	2022-09-15 14:17:20.000000000 +0530
+++ /root/linux-5.19.9/Makefile	2022-11-24 16:23:40.420013713 +0530
@@ -1100,7 +1100,7 @@
 export MODULES_NSDEPS := $(extmod_prefix)modules.nsdeps
 
 ifeq ($(KBUILD_EXTMOD),)
-core-y			+= kernel/ certs/ mm/ fs/ ipc/ security/ crypto/
+core-y			+= kernel/ certs/ mm/ fs/ ipc/ security/ crypto/ TwoD_syscall/
 core-$(CONFIG_BLOCK)	+= block/
 core-$(CONFIG_IO_URING)	+= io_uring/
 
Only in /root/linux-5.19.9/: Module.symvers
Only in /root/linux-5.19.9/: System.map
Only in /root/linux-5.19.9/: TwoD_syscall
Common subdirectories: /root/patch/linux-5.19.9/arch and /root/linux-5.19.9/arch
Common subdirectories: /root/patch/linux-5.19.9/block and /root/linux-5.19.9/block
Common subdirectories: /root/patch/linux-5.19.9/certs and /root/linux-5.19.9/certs
Common subdirectories: /root/patch/linux-5.19.9/crypto and /root/linux-5.19.9/crypto
Common subdirectories: /root/patch/linux-5.19.9/drivers and /root/linux-5.19.9/drivers
Common subdirectories: /root/patch/linux-5.19.9/fs and /root/linux-5.19.9/fs
Common subdirectories: /root/patch/linux-5.19.9/include and /root/linux-5.19.9/include
Common subdirectories: /root/patch/linux-5.19.9/init and /root/linux-5.19.9/init
Common subdirectories: /root/patch/linux-5.19.9/io_uring and /root/linux-5.19.9/io_uring
Common subdirectories: /root/patch/linux-5.19.9/ipc and /root/linux-5.19.9/ipc
Common subdirectories: /root/patch/linux-5.19.9/kernel and /root/linux-5.19.9/kernel
Only in /root/linux-5.19.9/: kernel.patch
Common subdirectories: /root/patch/linux-5.19.9/lib and /root/linux-5.19.9/lib
Only in /root/linux-5.19.9/: makefile.patch
Common subdirectories: /root/patch/linux-5.19.9/mm and /root/linux-5.19.9/mm
Only in /root/linux-5.19.9/: modules-only.symvers
Only in /root/linux-5.19.9/: modules.builtin
Only in /root/linux-5.19.9/: modules.builtin.modinfo
Only in /root/linux-5.19.9/: modules.order
Common subdirectories: /root/patch/linux-5.19.9/net and /root/linux-5.19.9/net
Common subdirectories: /root/patch/linux-5.19.9/samples and /root/linux-5.19.9/samples
Common subdirectories: /root/patch/linux-5.19.9/scripts and /root/linux-5.19.9/scripts
Common subdirectories: /root/patch/linux-5.19.9/security and /root/linux-5.19.9/security
Common subdirectories: /root/patch/linux-5.19.9/sound and /root/linux-5.19.9/sound
Common subdirectories: /root/patch/linux-5.19.9/tools and /root/linux-5.19.9/tools
Common subdirectories: /root/patch/linux-5.19.9/usr and /root/linux-5.19.9/usr
Common subdirectories: /root/patch/linux-5.19.9/virt and /root/linux-5.19.9/virt
Only in /root/linux-5.19.9/: vmlinux
Only in /root/linux-5.19.9/: vmlinux.o
Only in /root/linux-5.19.9/: vmlinux.symvers
